### DISCUSSION TITLE
taro如何先请求接口在加载页面

### DISCUSSION THREAD

**Discussion by zhengtan2003** [NervJS/taro#10670] [Q & A]
如题，因为小程序可以复制路径，这个预先请求接口不能放在首页。

**Comment by maopixin** [comment]
https://taro-docs.jd.com/taro/docs/optimized#%E8%B7%B3%E8%BD%AC%E9%A2%84%E5%8A%A0%E8%BD%BD
taro有一个preload，不知道是否符合你的业务
